### ISSUE TITLE
Popping up when ever it wants

### ISSUE THREAD

**Issue by AkashRyder** [will-stone/browserosaurus#390]
<img width="734" alt="Screen Shot 2021-04-18 at 23 00 30" src="https://user-images.githubusercontent.com/77179822/115146555-4fd99780-a09a-11eb-97e7-2da5c02bccc4.png">

I was on my desktop and browserosaurus kept popping up continuously (as in every few seconds). I've had to close the program down because it was hard to navigate to this webpage because it kept coming up. I've restarted it, reinstalled it and restarted my PC. 

In summary, it pops up when it's not wanted (when no links have been clicked on) and I cannot get it to stop. It had been working fine previously. I installed "Browserosaurus-darwin-x64-14.0.2" (which is the correct version for my device). I believe it to be the update that caused this issue as the previous update was fine.

**Comment by AkashRyder** [comment]
OH! The red dot is where I had my mouse resting.

**Comment by will-stone** [comment]
> OH! The red dot is where I had my mouse resting.

Are you saying this is no longer an issue?

If it is, firstly thanks for using Browserosaurus. From your picture it looks like it's popping up without a URL being sent to it, but there may be _something_ attempting to open your default browser. Or it's a bug with B, but certainly not one I've seen before. Which version of macOS are you using?

**Comment by AkashRyder** [comment]
I'm using the latest version of MacOS. 

When the program stopped working, I restarted my Mac etc and when it continued not to work I restored an old version of the program from Time Machine (the version of the program I had before it updated). It worked perfectly fine until it automatically updated to the version that was released a few days ago. It was there when I started to have problems. However, when I'm back on my computer, I'll try opening a link from a messenger app and see if that fixes it (since it seems to be trying to open a link that isn't there).

**Comment by will-stone** [comment]
Well, the way to test if it's an issue with another app is to see if it still happens when you set a browser other than B as your default.

Other than that, I'm not sure what there was in the last update to cause such an issue: https://github.com/will-stone/browserosaurus/compare/7bdc6ccf367766ad4bc6f7b372b0a7cd6263d9ef..b67fed9454818ad6d73286afc28dc7b496a8f9f8

**Comment by AkashRyder** [comment]
I tried it. I took a screen recording but it wouldn't let me send them here -- big files. I switched my deafult browser to firefox and then using B, i clciked make deafult. It said it made Safari deafult instead, yet when i clicked a link it acted as if i had B as my deafult browser??? Anyways, all the problems fixed themselves and it's working now. Only thing is it says Safari is the deafult browser and i can't choose B but in b's settings it doesn't say itneeds to be set as the deafult. It's acting as if it is the deafult so, that's weird but it works now!

**Comment by will-stone** [comment]
That's good. Yeah, the _not being able to see Browserosaurus as an option in the Default Browser list in System Preferences_ is a known issue; I and other devs our waiting for a fix from Apple: https://github.com/will-stone/browserosaurus/issues/317

Please take this kindly, and as its intended; this looks to be the first issue you've raised on an open-source project. I would highly suggest reading [An Open Source Etiquette Guidebook](https://css-tricks.com/open-source-etiquette-guidebook/#for-the-user). Essentially, you've logged an issue for an app that is provided for free, received almost instant free support, and not once used the words "please" or "thank you". Again, please take this as sage advice, we've all been at the beginning of open source at some point 😄

**Comment by AkashRyder** [comment]
I'm sorry about that. I was writing this late at night and then early in the morning-- I couldn't put words together well and I was a little nervous and all. I haven't really done one of these before. I highly appreciate your work here and your help was insanely fast and I thank you for that. I'm sorry if I came across as rude or anything, that wasn't my intention. Thank you again, have a lovely day!

**Comment by will-stone** [comment]
No worries. Like I said, we all start somewhere. Please let me know if you spot any more bugs 🙂
